pyproject.toml

```diff
@@ -1,15 +1,15 @@
 [tool.poetry]
 name = "taskiq-redis"
 version = "0.1.0"
-description = ""
-authors = ["taskiq-team"]
+description = "Redis integration for taskiq"
+authors = ["taskiq-team <taskiq@norely.com>"]
 packages = [{ include = "taskiq", from = "taskiq_redis" }]
 
 [tool.poetry.dependencies]
 python = "^3.7"
-taskiq = { git = "https://github.com/taskiq-python/taskiq", branch = "master" }
 redis = "4.3.4"
 flake8 = "^4.0.1"
+taskiq = {path = "../taskiq"}
```

Comment:
I've changed the dependency in the main branch. Now you don't have to specify a local folder path.